pkg/geo/geos/geos.go

```diff
@@ -127,6 +127,12 @@ func findLibraryDirectories(flagLibraryDirectoryValue string, crdbBinaryLoc stri
 		panic(err)
 	}
 	locs := []string{}
+	// Account for the libraries to be in a bazel runfile path.
+	if bazel.BuiltWithBazel() {
```

Comment:
actually, arguably after

```
if flagLibraryDirectoryValue != "" {
		locs = append(locs, flagLibraryDirectoryValue)
	}
```

is better since there's an explicit flag?